docs/journal.md

```diff
@@ -15,30 +15,58 @@
 
 14.10.2021
 
-- building and running VCOM project `Samples/Nucleo-TPM/VCOM`
+- Building and running VCOM project `Samples/Nucleo-TPM/VCOM`
 (Visual Studio 2017).
-- first successful communication with TPM core
-- investigation the root cause of response check failing
+- First successful communication with TPM core.
+- Investigation the root cause of response check failing.
 [link](https://github.com/lpn-plant/ms-tpm-20-ref/blob/master/Samples/Nucleo-TPM/VCOM/VCOM-TPM/VCOM-TPM.cpp#L198)
 
 15.10.2021
 
-- small fix for NDEBUG enabled compilation
-- protocol analysis
+- Small fix for NDEBUG enabled compilation.
+- Protocol analysis.
 
 18.10.2021
 
-- protocol analysis
+- Protocol analysis.
 
 19.10.2021
 
-- protocol analysis
-- decoding command and response data frames
-- setting up lib tpm2-tss and tpm2-pytss for future communication
+- Protocol analysis
+- Decoding command and response data frames.
+- Setting up lib tpm2-tss and tpm2-pytss for future communication.
 
-20.10.2021
+20/21.10.2021
 
-- protocol analysis
-- investigation of lib tpm2-tss and tpm2-pytss behaviour and wrong packet
-reception
+- Protocol analysis.
+- Investigation of lib tpm2-tss and tpm2-pytss behavior and wrong packet
+reception.
+
+26.10.2021
+
+- USB communication reworked, added circular buffer and functions responsible
+for command validation.
+- Development of python scripts responsible for serial communication and
+investigation the root cause of tpm2_tool @ /dev/ACM* failures.
+
+27.10.2021
+
+- Cleanup and refactor of python scripts.
+- Writing documentation.
```

Comment:
It would be great to leave some reference (link) or two for each day, For example, link to documentation MR (can be still WIP) or some useful external materials which were analyzed.

Reply:
Let's take of that in a separate MR.